script/migrations/02-upgrade_wallet.sol

```diff
@@ -12,6 +12,13 @@ import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 import "forge-std/console.sol";
 
+// Upgradeable version of KintoWallet
+contract KintoWalletV2 is KintoWallet {
+    constructor(IEntryPoint _entryPoint, IKintoID _kintoID)
+        KintoWallet(_entryPoint, _kintoID, IKintoAppRegistry(address(0)))
```

Comment:
shouldn't this receive a `_registry` var which is then passed to the constructor of the KintoWallet?

Reply:
these migrations were already executed and they are not going to be executed again so I want it to keep it consistent for the history that it was not passed.